### PR TITLE
PCHR-2707: Onboarding Wizard - Progress Bar Styling

### PR DESCRIPTION
**Pull request by deb1990**
## Overview
As part of this PR(and https://github.com/compucorp/org.civicrm.shoreditch/pull/55), the Progress Bar of the onboarding wizard, has been styled to match with the mock(https://projects.invisionapp.com/d/main#/console/10848582/229926109/preview).

### In Onboarding Form
welcome_to_civihr___hr16-7

### In User Edit page
admin___hr16-7

## Technical Details
1. Importing `SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/components/wizard` to reuse the css.
2. Created `_onboarding-wizard.scss` to center align the wizard.
3. Edited `webform-onboarding-progressbar.tpl.php` to implement the existing markup(https://github.com/civicrm/civihr/blob/8cc236aecbcdcfb52b5a3e31ca7069db529dada3/hrui/templates/CRM/HRUI/common/WizardHeader.tpl#L17-L17).

## Comments
Backstop tests are not added yet, those will be added as part of https://compucorp.atlassian.net/browse/PCHR-2708

---

### CODE REVIEW

File: civihr_default_theme/templates/node/webform-progressbar.tpl.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<?php
+/**
```
Comment (mickadoo) [inline civihr_default_theme/templates/node/webform-progressbar.tpl.php]: Wouldn't this mean that all webform progress bars would use the overridden template?

**Review by reneolivo — CHANGES_REQUESTED** [review]
please try to reuse code from this wizard component as much as possible:

* https://github.com/civicrm/org.civicrm.shoreditch/blob/926982a914cf6482562626219a71a434e34f4d79/scss/civicrm/common/_wizard.scss

**Comment by deb1990** [comment]
> please try to reuse code from this wizard component as much as possible:

@reneolivo I found that, there are 2 css available available for the wizard, the other one is https://github.com/compucorp/org.civicrm.shoreditch/blob/bd98a32ca66c44d57b6053d2f50499a58663e6e0/scss/bootstrap/components/_wizard.scss#L12-L12. And the same is used in `/civicrm/admin/leaveandabsences/import`, so I used that one here.

**Review by reneolivo — APPROVED** [review]

**Review by mickadoo — CHANGES_REQUESTED** [review]
@deb1990 just a reminder that the template should not be always overridden and you should copy your changes from the template file in this into [this file](https://github.com/compucorp/civihr-employee-portal-theme/blob/PCHR-2339-onboarding-wizard/civihr_default_theme/templates/contrib/webform-onboarding-progressbar.tpl.php)

**Pull request merged**